cli/coredump/coredump.go

```diff
@@ -0,0 +1,130 @@
+package coredump
+
+import (
+	"bytes"
+	"context"
+	"embed"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/codeclysm/extract"
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+//go:embed scripts/*
+var corescripts embed.FS
+
+func readEmbedFile(fs embed.FS, path string) (string, error) {
+	content, err := fs.ReadFile(path)
+	if err != nil {
+		return "", err
+	}
+	return string(content), nil
+}
+
+// packs coredump into a tar.gz archive.
+func pack(coreName string) error {
+	ctx, err := readEmbedFile(corescripts, "scripts/tarabart.sh")
```

Comment:
Please do not use the name `ctx`. It is now commonly used for `https://pkg.go.dev/context`.

Reply:
Fixed